package.json

```diff
@@ -1,36 +1,28 @@
 {
-  "version": "1.9.2",
   "devDependencies": {
-    "@fitbit/sdk": "^4.2.1",
-    "@fitbit/sdk-cli": "^1.7.3"
+    "@fitbit/sdk": "~5.0.0"
   },
   "fitbit": {
-    "appUUID": "19a118e0-3070-4f00-b76b-345d02853c70",
+    "appUUID": "c7514968-715d-48d3-927b-4b23c54e5f71",
     "appType": "clockface",
     "appDisplayName": "Roni",
     "iconFile": "resources/icon.png",
     "wipeColor": "",
     "requestedPermissions": [
       "access_activity",
       "access_heart_rate",
-      "access_internet"
+      "access_internet",
+	  "run_background"
     ],
     "buildTargets": [
-      "meson",
-      "gemini",
-      "mira"
+      "atlas",
+      "vulcan"
     ],
     "i18n": {
-      "en": {
+      "en-US": {
         "name": "Roni"
       }
-    }
-  },
-  "scripts": {
-    "build": "fitbit-build",
-    "cli": "npx fitbit"
-  },
-  "dependencies": {
-    "fitbit-google-analytics": "^1.2.0"
```

Comment:
Any reasons for removing the scripts?  I'd like to keep google analytics in the project.